Boolean.md

```diff
@@ -0,0 +1,84 @@
+##Boolean Type
+Boolean is a fancy term meaning something that is either true or false.  While numbers and math form the backbone of any piece of software, everything (and I do mean everything) in computers, software and microchips, breaks down to ones and zeros, true and false.  While it isn't important to understand why that is and how exactly it works, it is important to understand how boolean expressions work so that your code will be able to do something interesting like this:
+
+````Swift
+func finishCourse() {
+    if gradeIsPassing() {
+    	print("Hooray!")
+   } else {
+      print("Doh!")
+   }
+}
```

Comment:
I would like to show the students the `Bool` type in a more simple manner, not yet showing them its use case within a function that uses a conditional. I'm inclined to take this function out and go with something similar (or exact if you want to copy it) instead.

----

We know how to declare a variable of type `String`, `Int` and `Double` using type inference like so:

```swift
let name = "Luke"
let age = 34
let lengthOfLightSaber = 32.5

// name is a constant of type String with a value of "Luke"
// age is a constant of type Int with a value of 34
// lengthOfLightSaber is a constant of type Double with a value of 32.5
```

But.. what if we want to declare a variable as being true or false (type `Bool`).

```swift
let hasTheForce = true
// hasTheForce is a constant of type `Bool` with a value of true
```

Option clicking this constant in the playground file to see its type will show us this result:

![result](http://i.imgur.com/C5c1KTI.png?1)

Here is another example, except here we're creating a variable which will allow us to change the value.

```swift
var isHungry = false
// isHungry is a variable of type Bool with a value of false

isHungry = true
// isHungry now has a value of true
```